TitleBarCustomization/explainer.md

```diff
@@ -11,8 +11,8 @@ This document is intended as a starting point for engaging the community and sta
 
 ## Table of Contents
  - [Introduction](#introduction)
- - [Examples of title bar customization on desktop apps](#examples-of-title-bar-customization-on-desktop-apps)
- - [Problem to solve: PWA Title bar area is system reserved](#problem-to-solve-pwa-title-bar-area-is-system-reserved)
+ - [Examples of desktop apps customizing the title bar area](#examples-of-desktop-apps-customizing-the-title-bar-area)
+ - [Problem to solve: Desktop installed web apps title bar area is system reserved](#problem-to-solve-installed-desktop-web-apps-title-bar-area-is-system-reserved)
  - [Goals](#goals)
  - [Proposal](#proposal)
    - [Overlaying Window Controls](#overlaying-window-controls)
```

Comment:
This one still needs to be updated to match the new heading text, but all other links work :)

Reply:
Done